ansible/templates/minigraph_cpg.j2

```diff
@@ -1,6 +1,7 @@
   <CpgDec>
     <IsisRouters xmlns:a="http://schemas.datacontract.org/2004/07/Microsoft.Search.Autopilot.Evolution"/>
     <PeeringSessions>
+{% if cardtype is not defined or cardtype != 'supervisor' %}
```

Comment:
In PR https://github.com/Azure/sonic-mgmt/pull/3218, I used 'card_type', instead of 'cardtype'. Could you please change 'cardtype' to 'card_type' in this PR as well.

Reply:
sure